td-layout/src/memslice.rs

```diff
@@ -95,10 +97,15 @@ pub unsafe fn get_dynamic_mem_slice_mut<'a>(t: SliceType, base_address: usize) -
             base_address as *const u8 as *mut u8,
             TD_PAYLOAD_EVENT_LOG_SIZE as usize,
         ),
+        SliceType::RelocatedMailbox => core::slice::from_raw_parts_mut(
```

Comment:
Please also add test cases to cover this new area.

Reply:
Sure, thanks for reminding